### PR TITLE
Add missing tests to the VMS test scripts

### PR DESCRIPTION
**Pull request by levitte**
Thanks to Douglas Fyfe @ VSI for making me aware of this

### CODE REVIEW

File: test/maketests.com [hunk 1]
```diff
@@ -150,7 +150,8 @@ $ TEST_FILES = "BNTEST,ECTEST,ECDSATEST,ECDHTEST,IDEATEST,"+ -
 	       "EVP_TEST,EVP_EXTRA_TEST,IGETEST,JPAKETEST,SRPTEST,"+ -
 	       "ASN1TEST,V3NAMETEST,HEARTBEAT_TEST,"+ -
 	       "CONSTANT_TIME_TEST,VERIFY_EXTRA_TEST,"+ -
-               "CLIENTHELLOTEST,SSLV2CONFTEST,DTLSTEST"
+               "CLIENTHELLOTEST,SSLV2CONFTEST,DTLSTEST,"+ -
+               "BAD_DTLS_TEST,FATALERRTEST"
```
Comment (richsalz) [inline test/maketests.com]: while you're in the area, it would be nice to update the comment at line 155 to remove outdated test names.

**Review by richsalz — APPROVED** [review]

File: test/maketests.com (hunk 1 above)
Comment (levitte) [reply test/maketests.com]: Uhmmmm...  I'm thinking I should remove that line, is that what you mean?

Comment (levitte) [reply test/maketests.com]: I decided to do so.  Reconfirm?

Comment (richsalz) [reply test/maketests.com]: reconfirm.  I just wanted the outdated comment fixed, in whatever was most appropriate :)

**Comment by levitte** [comment]
Merged.  ceee07cee5d7f07f1b48798ba8fe790204644b9e and 5e7cd29c95767d214cc98bc7adfb91c6271ac87a

**Pull request closed without merging**